### ISSUE TITLE
full_slug field on stories is inconsistent with trailing slashes

### ISSUE THREAD

**Issue by fredrikekelund** [storyblok/storyblok#711]
**The feature would affect:**
- [ ] *app.storyblok.com (CMS - Interface)*   
- [x] *api.storyblok.com (CMS - Content Delivery API)*
- [x] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**
When fetching stories from the Content Delivery or Management API, the `full_slug` field is inconsistent with trailing slashes. For folder startpages, there is a trailing slash, but for all other pages, there is not.

**Describe the solution you'd like**
It would be very helpful to at least have this mentioned in the docs. Updating the current API might not be realistic since it could break existing integrations, but in potential future versions of the API, I think it would be more useful to be consistent. There is already the `is_startpage` field to determine whether a particular page is a startpage or not.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by LukasHechenbergerID** [comment]
Is there any advancement with this issue?

**Comment by IzakJackson** [comment]
Is there any workarounds for this?

**Comment by LukasHechenbergerID** [comment]
@IzakJackson yes, you can just check if there is a trailing slash like that and remove it:
`
if (link.linktype === 'story' && link.cached_url) {
        // Remove trailing slash
        if (link.cached_url.endsWith('/')) {
            modifiedLink = link.cached_url.slice(0, -1);
        }
        return `/${modifiedLink}`;
`

**Comment by DaveZIvy** [comment]
I've got the same problem, and am currently removing the slashes manually in all of our different blocks that contain a link, but having an option in settings for this in storyblok would be a big help.

I've tried to update our code that makes the api call to get all the data for the page, but it threw enough errors that it was easier to just update the code for the links.

Please add a setting to include / exclude trailing slashes.

**Comment by alvarosabu** [comment]
Hi @fredrikekelund, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
